server/middleware/setUpWebSession.ts

```diff
@@ -1,19 +1,23 @@
 import { v4 as uuidv4 } from 'uuid'
-import session from 'express-session'
+import session, { Store } from 'express-session'
 import RedisStore from 'connect-redis'
 import express, { Router } from 'express'
 import { createRedisClient } from '../data/redisClient'
 import config from '../config'
 import logger from '../../logger'
 
 export default function setUpWebSession(): Router {
-  const client = createRedisClient()
-  client.connect().catch((err: Error) => logger.error(`Error connecting to Redis`, err))
+  let store: Store
```

Comment:
should this fall back to the inmemory store?  what happens if you pass undefined?

Reply:
Express defaults to a MemoryStore, so passing undefined means it uses its own in-memory store for session data.